NeuroStrip.py

```diff
@@ -16,23 +16,42 @@
 from slicer.ScriptedLoadableModule import *
 from slicer.util import VTKObservationMixin
 
-try:
-    import neurostrip
-except ImportError:
-    slicer.util.pip_install("neurostrip[cpu]")
-    import neurostrip
 
+#
+# Dependencies
+#
+
+
+def checkPythonDependencies() -> bool:
+    try:
+        from neurostrip import predict
+    except ImportError:
+        return False
+    return True
+
+
+def installPythonDependencies() -> None:
+    msg = "Extra python packages (neurostrip and onnxruntime) are required."
+    msg += "\nClick OK to install them for NeuroStrip."
+    if not slicer.util.confirmOkCancelDisplay(msg):
+        logging.info('Installation of neurostrip aborted by user')
+        return
+    slicer.util.pip_install("neurostrip[cpu]")
+    
+    msg = "A restart is required"
+    msg += "\nClick OK to restart 3D Slicer"
+    if not slicer.util.confirmOkCancelDisplay(msg):
+        logging.info('Restart of 3D Slicer aborted by user')
+        return
+    slicer.util.restart()
```

Comment:
Is a restart really required here?  Usually we can just `import` the package after install and keep going (in fact, that's the way it worked before I believe).

Reply:
resolved in this PR: https://github.com/dyollb/SlicerNeuroStrip/pull/4